cypress/integration/AdvancedSearchPage/KeywordsSection.feature

```diff
@@ -0,0 +1,73 @@
+        And trial info displays "Results 1-10  of 179 for your search "
+        And the criteria table displays the following
+           | Category | Selection |
+           | Keywords/Phrases      | psa       |
+        And the url query has the following corresponding code   
+        | parameter | value                        |
+        | q      | psa|
+        | loc       | 0                            |
+        | rl        | 2                            |
+
+    Scenario: User has an option to search by phrase 
+        Given the user navigates to "/about-cancer/treatment/clinical-trials/search/advanced"
+        Then the page title is "Find NCI-Supported Clinical Trials"
+        And "Keywords/Phrases" form section is displayed
+        When user types "chronic kidney disease due to hypertension" in "KeywordsPhrases" field
+        When user clicks on "Find Trials" button
+        Then the search is executed and results page is displayed
+        And trial info displays "Results 1-1  of 1 for your search "
+        And the criteria table displays the following
+           | Category | Selection |
+           | Keywords/Phrases      | chronic kidney disease due to hypertension       |
+        And the url query has the following corresponding code   
+        | parameter | value                        |
+        | q      | chronic kidney disease due to hypertension|
+        | loc       | 0                            |
+        | rl        | 2                            |
+
+    Scenario: Negative : User searches by keyword that does not exist
+        Given the user navigates to "/about-cancer/treatment/clinical-trials/search/advanced"
+        Then the page title is "Find NCI-Supported Clinical Trials"
+        And "Keywords/Phrases" form section is displayed
+        When user types "penguin" in "KeywordsPhrases" field
+        When user clicks on "Find Trials" button
+        Then the search is executed and no results page is displayed
+        And the criteria table displays the following
+        | Category | Selection |
+        | Keywords/Phrases      | penguin        |
+        And the url query has the following corresponding code  
+        | parameter | value                        |
+        | q      | penguin|
+        | loc       | 0                            |
+        | rl        | 2                            |
+
+    Scenario: User has an option to go to search results url directly and modify search
+            Given the user navigates to "/about-cancer/treatment/clinical-trials/search/r?loc=0&q=chronic%20kidney%20disease%20due%20to%20hypertension&rl=2"
+            Then the page title is "Clinical Trials Search Results"
+            And trial info displays "Results 1-1  of 1 for your search "
+            When user clicks on Modify Search Criteria button
+            Then "KeywordsPhrases" field has value "chronic kidney disease due to hypertension" 
+
```

Comment:
Add search refine scenario after navigating to url direcftly by adding the following steps aftet line 72
```
When user clears "KeywordsPhrases" input  field
 When user types "psa" in "KeywordsPhrases" field
        When user clicks on "Find Trials" button
        Then the search is executed and results page is displayed
        And trial info displays "Results 1-10  of 179 for your search "
        And the criteria table displays the following
           | Category | Selection |
           | Keywords/Phrases      | psa       |
        And the url query has the following corresponding code   
        | parameter | value                        |
        | q      | psa|
        | loc       | 0                            |
        | rl        | 2                            |

```